src/common/types/config.ts

```diff
@@ -1,5 +1,35 @@
 import { Server } from './server';
 
+/**
+ * This interface documents the sqlectron.json file. The only thing guaranteed to
+ * exist is the `servers` key/value pair (instantiated to empty array). The rest
+ * will only exist if the user goes into the Settings modal and hits save.
+ */
+export interface ConfigFile {
```

Comment:
@MasterOdin wondering if wouldn't be better to make `Config` extend `ConfigFile` to avoid duplicated fields, event though most base fields are optional.